grids/graph.go

```diff
@@ -291,21 +291,30 @@ func (ggraph *Graph) DelVertex(id string) error {
 			_, eid, sid, did, label := SrcEdgeKeyParse(skey)
 			ekey := EdgeKey(ggraph.graphKey, eid, sid, did, label)
 			delKeys = append(delKeys, skey, ekey)
+
+			edgeID := ggraph.kdb.keyMap.GetEdgeID(ggraph.graphKey, eid)
+			ggraph.kdb.keyMap.DelEdgeKey(ggraph.graphKey, edgeID)
 		}
 		for it.Seek(dkeyPrefix); it.Valid() && bytes.HasPrefix(it.Key(), dkeyPrefix); it.Next() {
 			dkey := it.Key()
 			// get edge ID from key
 			_, eid, sid, did, label := SrcEdgeKeyParse(dkey)
 			ekey := EdgeKey(ggraph.graphKey, eid, sid, did, label)
 			delKeys = append(delKeys, ekey)
+
+			edgeID := ggraph.kdb.keyMap.GetEdgeID(ggraph.graphKey, eid)
+			ggraph.kdb.keyMap.DelEdgeKey(ggraph.graphKey, edgeID)
 		}
 		return nil
 	})
 
+	ggraph.kdb.keyMap.DelVertexKey(ggraph.graphKey, id)
+
 	return ggraph.kdb.graphkv.Update(func(tx kvi.KVTransaction) error {
 		if err := tx.Delete(vid); err != nil {
 			return err
 		}
+		ggraph.kdb.keyMap.DelVertexKey(ggraph.graphKey, id)
```

Comment:
error needs to be handled

Reply:
Added error output

Reply:
The DelVertexKey error is still not handled